app/controllers/admin/conference_controller.rb

```diff
@@ -174,8 +174,25 @@ def edit
       end
     end
 
+    def conference_info
+      # To display sponsors in the conference wide information page
+      @sponsors = @conference.sponsors
+      @current_event_schedules = @program.selected_schedule.event_schedules.current
```

Comment:
If there is no selected_schedule, the page breaks. 
Even though we expect the conference to have a selected_schedule when the conference takes place, it would be nice for the page to be able to successfully load at all times.